src/scss/index.scss

```diff
@@ -1 +1,64 @@
+@import "_variables.scss";
+@import "_navbar.scss";
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
 
+.bold {
+  font-weight: 900;
+}
+
+.log-in-bg {
+  background-color: $primary;
+}
+
+.sign-up-menu {
+  background-color: $info-dark;
+  // border-radius: 20px 0 0 20px;
+}
+
+.log-in-menu {
+  background-color: $info;
+  // border-radius: 0 20px 20px 0;
+}
+
+.log-in-text {
+  color: $secondary;
+}
+
+.btn {
+  background-color: $button-bg;
+}
+
+.btn:hover {
+  background-color: $button-bg-hover;
+}
+
+.profile-image {
+  background-image: url(/images/selfie.jpg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 20vw;
+  width: 20vw;
+}
+
+.user-image {
+  background-image: url(/images/profile-image.jpeg);
+  background-position: center;
```

Comment:
DRY principle

Reply:
Is it better to repeat classes in the scss file than repeating values? like adding .profile-image, .user-image, .user-imag1 {} as one then .user-image, .user-image-1 {} as a seperate?

Reply:
It would be best to create a utility class and use this in the HTML:

```css
.bg-image-center {
  background-position: center;
  background-size: cover;
  background-repeat: no-repeat;
}

.user-image {
  background-image: url(/images/profile-image.jpeg);
}
```

```html
<div class="user-image bg-image-center">
```

Reply:
You could also use some SASS magic:

```scss
.user-image {
  @extend .bg-image-center;
}
```